server/templates/base.html

```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1"/>
     <title>Dearborn Coding Club</title>
+    < logo image here ></logo>
```

Comment:
I'm not too sure if anything would actually display, since I don't think that logo image here is a valid tag. I think we should just add a span tag.